WebRoot/static/dhtml/company.js

```diff
@@ -3,22 +3,32 @@
  * https://usefulangle.com/post/118/javascript-intersection-observer
  */
 
-var panel = document.getElementById("company-panel");
-var summary = document.getElementById("mobile-summary");
+const panel = document.getElementById("company-panel");
+const edit = document.getElementById("edit");
+const summary = document.getElementById("mobile-summary");
```

Comment:
`const` comes from ES6 so always use `var` to stay compatible with ES5 (see CONTRIBUTING.md -> JavaScript)

Reply:
Maybe we should use ES6 🙊 it came out in 2015! It is just that most tutorials I find uses ES6